modules/log6x-6-2-1-rn.adoc

```diff
@@ -0,0 +1,41 @@
+
+* Before this update, issuing the `oc explain obsclf.spec.filters` command did not list all the supported filters in the command output. With this update, all the supported filter types are listed in the command output. (link:https://issues.redhat.com/browse/LOG-6753[LOG-6753])
+
+
+* Before this update the log collector flagged a `ClusterLogForwarder` resource with multiple inputs to a LokiStack output as invalid due to  incorrect internal processing logic. This update fixes the issue. (link:https://issues.redhat.com/browse/LOG-6758[LOG-6758])
+
+* Before this update, issuing the `oc explain` command for the `clusterlogforwarder.spec.outputs.syslog` resource returned an incomplete result. With this update, the missing supported types for `rfc` and `enrichment` attributes are listed in the result correctly. (link:https://issues.redhat.com/browse/LOG-6869[LOG-6869])
+
+* Before this update, empty OpenTelemetry (OTEL) tuning configuration caused validation errors. With this update, validation rules have been updated to accept empty tuning configuration. (link:https://issues.redhat.com/browse/LOG-6878[LOG-6878])
+
+* Before this update the {clo} could not update the `securitycontextconstraint` resource that is required by the log collector. With this update, the required cluster role has been provided to the service account of the {clo}. As a result of which, {clo} can create or update the  `securitycontextconstraint` resource. (link:https://issues.redhat.com/browse/LOG-6879[LOG-6879])
+
+* Before this update, the API documentation for the  URL attribute of the `syslog` resource incorrectly mentioned the value `udps` as a supported value. With this update, all references to `udps` have been removed. (link:https://issues.redhat.com/browse/LOG-6896[LOG-6896])
+
+* Before this update, the {CLO} was intermittently unable to update the object in logs due to update conflicts. This update resolves the issue and prevents conflicts during object updates by using the `Patch()` function instead of the `Update()` function. (link:https://issues.redhat.com/browse/LOG-6953[LOG-6953])
+
+* Before this update, Loki ingesters that got into an unhealthy state due to networking issues stayed in that state even after the network recovered. With this update, you can configure  the {loki-op} to perform service discovery more often so that unhealthy ingesters can rejoin the group. (link:https://issues.redhat.com/browse/LOG-6992[LOG-6992])
+
+* Before this update, the Vector collector could not forward Open Virtual Network (OVN) and Auditd logs. With this update, the Vector collector can forward OVN and Auditd logs. (link:https://issues.redhat.com/browse/LOG-6997[LOG-6997])
+
+
+[id="logging-release-notes-6-2-1-CVEs_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2022-49043[CVE-2022-49043]
```

Comment:
There isn't any CVEs in 6.2.1 after I attached the new build. can you update this?

Reply:
thanks, removed